### ISSUE TITLE
Section nav: incorrect heading level

### ISSUE THREAD

**Issue by mattsharkey** [web-illinois/toolkit#231]
The heading for the left nav must be an H2, not an H3. All new sections / regions must begin with H2.

* The "In This Section" heading could remain in the page but be hidden visually to correct this issue.

**Comment by mattsharkey** [comment]
The "In This Section" heading is hidden visually, so this should be fixed.
